cubeviz/data_factories/__init__.py

```diff
@@ -98,6 +111,10 @@ def load_data(self, data_filenames):
                         # The data must be floating point as spectralcube is expecting floating point data
                         data.add_component(component=hdu.data.astype(np.float), label=component_name)
 
+                        if 'BUNIT' in hdu.header:
```

Comment:
It might be useful to have a comment here about what `BUNIT` means in the FITS standard (or even just a link:https://heasarc.gsfc.nasa.gov/docs/fcg/standard_dict.html)